Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Provis.Core.DTO.userDTO
+{
+    public class UserActiveInviteDTO
```

Comment:
Can this model be populated and extending by some user cases or we can just return true/false at controller